octobot_trading/exchanges/exchange_builder.py

```diff
@@ -43,25 +43,30 @@ def __init__(self, config, exchange_name):
     Build
     """
     async def build(self):
-        await self.exchange_manager.initialize()
+        try:
```

Comment:
To be cleaner, can `async def build(self)` call an `async def _build(self)` as followed ? 
```
async def build(self):
        try:
                await self._build()
        except Exception as e:
            # stop exchange manager if an exception occurred when building it
            await self.exchange_manager.stop()
            raise e        
```
With  `async def _build(self)` that contains the previous  `async def build(self)` code.